### ISSUE TITLE
Background color

### ISSUE THREAD

**Issue by LondonJammo** [lukepighetti/salomon_bottom_bar#32]
Currently have to use a workaround with nested containers etc in order to allow setting a background color.

However `BottomNavigationBar` allows this:

```
BottomNavigationBar(

...
[Color](https://api.flutter.dev/flutter/dart-ui/Color-class.html)? fixedColor,
[Color](https://api.flutter.dev/flutter/dart-ui/Color-class.html)? backgroundColor,
...
)
```

**Comment by lukepighetti** [comment]
Feel free to open a PR :)

**Comment by Abh1shekShinde** [comment]
can I try to contribute..
If there is any other thing that needs contribution ..then i can try to help in contributing

**Comment by lukepighetti** [comment]
This appears to be the only open issue @Abh1shekShinde

**Comment by sevenfold4** [comment]
I am trying to change the background color as well through ThemeData and
bottomNavigationBarTheme, but it doesnt seem to respond, am i doing something wrong here?

**Comment by lukepighetti** [comment]
closed by #44

**Comment by lukepighetti** [comment]
Feature now available on `3.3.2` on pub
